usaspending_api/database_scripts/broker_matviews/award_matview.sql

```diff
@@ -0,0 +1,1880 @@
+drop materialized view if exists award_matview;
```

Comment:
This is problematic for materialized views based upon this materialized view. Either you couldn't drop it, or you would need to do a cascading drop which would delete the API matviews created from this as well.

Reply:
Agreed. This was fine for testing but we'll need a better way to manage these once other dependencies are incorporated. We can leave it in for now and handle this iteratively as we add the dependencies.

Reply:
I would have agreed with you two months ago. After wrestling with my own matview creations, I think it would be better to make this change earlier rather than later. 

What I had to do was create the new materialized view with a temporary name. After it is created the existing matview is renamed and the new matview name is changed from the temporary name to the desire final name.